### PR TITLE
test: Add test for getblockheader verboseness

### PR DESCRIPTION
**Pull request by torhte**
Improve test coverage by adding a test for getblockheader with verbose argument set to false.

### CODE REVIEW

File: test/functional/rpc_blockchain.py [hunk 1]
```diff
@@ -280,6 +280,14 @@ def _test_getblockheader(self):
         assert isinstance(int(header['versionHex'], 16), int)
         assert isinstance(header['difficulty'], Decimal)
 
+        # Test with verbose=False, which should return the header as hex.
+        header_hex = node.getblockheader(blockhash = besthash, verbose = False)
```
Comment (theStack) [inline test/functional/rpc_blockchain.py:284]: It is preferred to pass named arguments without any spaces between the `=`. See https://www.python.org/dev/peps/pep-0008/#other-recommendations (point 5) and the discussion https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value.

**Review by theStack — COMMENTED** [review]
Concept ACK 
Welcome as a new contributor torhte! 🎉

File: test/functional/rpc_blockchain.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
     TIME_GENESIS_BLOCK,
 )
 from test_framework.messages import (
-    msg_block,
+    msg_block, FromHex, CBlockHeader,
```
[the remark this answers is not in the document]
Comment (torhte) [reply test/functional/rpc_blockchain.py]: Thanks!

File: test/functional/rpc_blockchain.py (hunk 1 above)
Comment (torhte) [reply test/functional/rpc_blockchain.py]: Thanks!

**Review by theStack — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/19722/commits/5067c5acc30c5cf87496c1bf8eb03712cc66b206

**Pull request merged**